docs/pingfunctionality.md

```diff
@@ -1,51 +1,27 @@
-# Mashling CLI Pingfunctionality
+# Mashling Pingfunctionality
```

Comment:
Why not get rid of this doc file completely and just collapse these ping docs into the primary gateway/README.md docs? At this point it is just a command line argument to the default gateway command.